data/categories.txt

```diff
@@ -68,7 +68,7 @@ sk:Jedlo
 @transport
 en:Transport
 ru:Транспорт
-ar:النقل
+ar:مواصلات|نقل
```

Comment:
They are not needed but they won't hurt if we add more translations later.
The main reason for adding them was to make sure that group translations (starting with '@') don't break whatever includes them.